shared/js/ui/templates/site.es6.js

```diff
@@ -1,7 +1,7 @@
 const bel = require('bel')
 const toggleButton = require('./shared/toggle-button.es6.js')
 const ratingHero = require('./shared/rating-hero.es6.js')
-const trackerNetworksIcon = require('./shared/tracker-network-icon.es6.js')
+const trackerNetworksHeroIcon = require('./shared/tracker-network-hero-icon.es6.js')
```

Comment:
What's the background of this change? I'm not sure adding `hero` to the name is better. It's used on the main popup, after all.